compose/production/django/start

```diff
@@ -6,6 +6,6 @@ set -o nounset
 
 
 python /app/manage.py collectstatic --noinput
-
+python manage.py migrate
```

Comment:
Do we need this?

With a production database, I don't generally like to run a `migrate` automatically since it can really jack up your database if you do it accidentally...

Reply:
agree. I guess it would be a good idea to document that it doesn't do that automatically somewhere since it differs from the local.yml behavior and I was confused why everything was 500ing for a while

Reply:
yes! I agree. The documentation (just merged to master) has a section with this information, but I forgot to add it. Maybe add it under item 4 in `docs/getting-started/install.rst`?